model/pipeline/src/main/proto/standard_window_fns.proto

```diff
@@ -25,7 +25,7 @@ syntax = "proto3";
 
 package org.apache.beam.model.pipeline.v1;
 
-option go_package = "pipeline_v1";
+option go_package = "github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1;pipeline_v1";
```

Comment:
not really a go expert, is this valid? Or did you replace the string incorrectly?

Reply:
This is valid, but it's not go, it's proto option syntax. Unfortunately it's only used in places and never fully described AFAICT and I don't want to dig into the C++ source to find out.

You can see an example of it here, and search for "go_package": https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.descriptor.pb?hl=en

Reply:
If you look bit further you can see the actual protocol buffer field for go_package (cus apparently Protos are self describing on multiple levels).

Part of the reason this change is so big is because in the last year the implementation of Go protocol buffers changed dramatically, and apparently they're moving to recommending the full package path for go_packages now. This is overall good since it means it will be able to work with Go Modules eventually. Who knows, Maybe it eventually saves us from needing to regenerate them.